content/post/council-meeting-2022-07-18/chart.csv

```diff
@@ -1,26 +1,25 @@
+https://a2gov.legistar.com/LegislationDetail.aspx?ID=5717312&GUID=9D1C2879-DCA9-46BE-82E2-7744AF5F31D2&G=505481FE-B2ED-4B34-9A92-FDB26536D1E1&Options=&Search=,consent pass,CA-5,"Resolution to Approve Street Closure of Washington Street between Fletcher and Thayer Streets for the University of Michigan Go Blue Mix on Saturday, September 3, 2022 from 4:00 PM until 2:00 AM on Sunday, September 4, 2022",TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,Absent,TRUE,TRUE,TRUE
+,consent pulled amendment,CA-6,"Resolution to Approve Street Closings for the “Dancing in the Streets” Special Event - September 4, 2022 (Deleted from Agenda 7/15/22)",,,,,,,,,,,
+https://a2gov.legistar.com/LegislationDetail.aspx?ID=5717305&GUID=33D4D5E0-631A-438D-9A60-8B95E1BEBCA8&G=505481FE-B2ED-4B34-9A92-FDB26536D1E1&Options=&Search=,consent pass,CA-7,"Resolution to Approve an Agreement with the Washtenaw County Sheriff’s Office for Weapons Screening Services in the Ann Arbor Justice Center (NTE $205,000.00)",TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,Absent,TRUE,TRUE,TRUE
+https://a2gov.legistar.com/LegislationDetail.aspx?ID=5717307&GUID=374A4703-4D25-42DB-98EB-145A1B79A439&G=505481FE-B2ED-4B34-9A92-FDB26536D1E1&Options=&Search=,consent pass,CA-8,"Resolution to Approve Amendment No. 2 to the Professional Services Agreement with Fishbeck, for Water Treatment Professional Engineering Services ($400,000.00 increase, total contract $1,250,000.00)",TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,Absent,TRUE,TRUE,TRUE
+https://a2gov.legistar.com/LegislationDetail.aspx?ID=5717308&GUID=00EBF9C1-4C9F-457F-8404-07DEBB243012&G=505481FE-B2ED-4B34-9A92-FDB26536D1E1&Options=&Search=,consent pass,CA-9,"Resolution to Authorize a Purchase Order to Carrier & Gable, Inc. for Traffic Control Materials and Supplies ($640,500.00)",TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,Absent,TRUE,TRUE,TRUE
+https://a2gov.legistar.com/LegislationDetail.aspx?ID=5717310&GUID=82E7ED3F-00CD-4F48-985C-5C89688C182B&G=505481FE-B2ED-4B34-9A92-FDB26536D1E1&Options=&Search=,consent pass,CA-10,"Resolution to Authorize a Sole Source Purchase Order to Yunex LLC for Siemens Traffic Control Products in the Amount of $690,000.00",TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,Absent,TRUE,TRUE,TRUE
+https://a2gov.legistar.com/LegislationDetail.aspx?ID=5724093&GUID=25F5232B-1EC0-47FE-9F56-4BEBF4B3FAAC&G=505481FE-B2ED-4B34-9A92-FDB26536D1E1&Options=&Search=,consent pass,CA-12,"Resolution to Purchase a Water Main Easement at 2311 E. Stadium ($119,896.00) (8 Votes Required)",TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,TRUE,Absent,TRUE,TRUE,TRUE
+https://a2gov.legistar.com/LegislationDetail.aspx?ID=5717306&GUID=24D95CB5-6E7C-434B-A7FF-4EC36A311E1E&G=505481FE-B2ED-4B34-9A92-FDB26536D1E1&Options=&Search=,consent pulled pass,CA-11,Resolution to Prohibit On-Street Parking on Both Sides of Barton Drive from Northside Avenue to Pontiac Trail,TRUE,TRUE,FALSE,TRUE,TRUE,TRUE,TRUE,Absent,TRUE,TRUE,TRUE
```

Comment:
this confused the css because of "pulled": I used pulled to indicate that an agenda item was removed from the agenda, @akgood used pulled to indicate what they say during the meeting—that they pulled it from the CA to do a rollcall vote. I said "rollcall" when that happened in my csv.

Reply:
But sometimes, a consent agenda item might get "pulled" for discussion, but then they approve it unanimously anyway, so it's not a rollcall... ;)